pkg/runner/runner.go

```diff
@@ -255,3 +261,23 @@ func Run(cfg *Config) error {
 	}
 	return err
 }
+
+func registerTelemetryMetrics(client *pgclient.Client) {
```

Comment:
Consider better naming: eg `startTelemetryEngine` since this code is starting the engine and registering few metrics...

Reply:
Made this `initTelemetryEngine` since starting here only and stopping in caller doesn't look nice. Hence, it initializes and registers metrics and start and deferred stop is done in the caller.